### PR TITLE
id: 76-disabled-users-can-t-login

### PR DESCRIPTION
**Pull request by jovannypcg**
Add validation when disabled users try to login.

### CODE REVIEW

File: src/main/groovy/com/nearsoft/happinessometer/web/controller/LoginController.groovy [hunk 1]
```diff
@@ -75,6 +75,19 @@ class LoginController extends BaseController {
                     redirectAttributes.addFlashAttribute("login", login)
                     return "redirect:/login"
                 }
+
+                if (isUnathorized(ex)) {
```
Comment (abadongutierrez) [inline src/main/groovy/com/nearsoft/happinessometer/web/controller/LoginController.groovy]: isUnathorized and isForbidden does the same thing. I think they could be in the same if with an ||

Comment (jovannypcg) [inline src/main/groovy/com/nearsoft/happinessometer/web/controller/LoginController.groovy]: Done, it got reduced to only one if statement :)

Comment (abadongutierrez) [inline src/main/groovy/com/nearsoft/happinessometer/web/controller/LoginController.groovy]: thanks

**Comment by abadongutierrez** [comment]
@jovannypcg please "squash and merge" this changes

**Comment by abadongutierrez** [comment]
👍

**Pull request merged**